onnx/numpy_helper.py

```diff
@@ -318,10 +313,9 @@ def to_optional(optional: OptionalProto) -> Optional[Any]:
     elem_type = optional.elem_type
     if elem_type == OptionalProto.UNDEFINED:
         return opt
-    value_field = mapping.OPTIONAL_ELEMENT_TYPE_TO_FIELD[elem_type]
-    value = getattr(optional, value_field)
+    value = helper.get_attr_from_optional_elem_type(optional, elem_type)
     # TODO: create a map and replace conditional branches
-    if elem_type == OptionalProto.TENSOR or elem_type == OptionalProto.SPARSE_TENSOR:
+    if elem_type in (OptionalProto.TENSOR, OptionalProto.SPARSE_TENSOR):
         opt = to_array(value)
     elif elem_type == OptionalProto.SEQUENCE:
         opt = to_list(value)
```

Comment:
Change to `return to_list(optional.tensor_value)`

Reply:
Likewise for all the cases. We don't need the helper utility method, whose purpose is only to do this switch/case anyway.

Reply:
Done. Thanks!